dask/utils.py

```diff
@@ -498,6 +498,25 @@ def wrapper(method):
             if doc is None:
                 doc = ''
 
+            # Insert disclaimer near the top, after the title.
+            if doc:
+                copied = (
+                    "This docstring was copied from %s.%s.%s\n"
+                    "Some inconsistencies with the Dask version may exist. "
+                    % (original_klass.__module__, original_klass.__name__,
+                        method_name)
```

Comment:
This doesn't indent the warning to the same level as the original docstring. See comment below for solution: